src/TriviaGenerator.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState } from 'react';
+import { useCollapse } from 'react-collapsed';
```

Comment:
Just realized you aren't even using this import 😄  You managed to implement the collapse feature using just JavaScript, ha. Remove this line and also remove the dependency from package.json (I'll put a comment there as well).